Umbraco-Cloud/Upgrades/Migrating-from-8-to-10/index.md

```diff
@@ -120,7 +120,113 @@ Read more about these changes in the [IPublishedContent section of the Documenta
 
 Depending on the size of the project that is being migrated and the amount of custom code and implementations, this step is going to require a lot of work.
 
-## Step 4: Deploy and Test on Umbraco Cloud
+## Step 4: Member Migration
+
+If you have Members in your Umbraco database these were not transferred when you restored the content from your Cloud environment. You have to migrate them manually using database queries. 
+
+* Open your both your Umbraco 8 database and your Umbraco 10 database in SQL Server Management Studio. Connection details for your database are available in the Cloud Portal.
+
+* Your Umbraco 10 database should not have any Members or Member Groups at this point, but if it does you can delete them all with the following queries:
+
+    ```
+	DELETE FROM umbracoPropertyData WHERE versionid in (SELECT id from umbracoContentVersion WHERE nodeId IN (SELECT nodeId FROM cmsMember))
+	DELETE FROM umbracoContentVersion WHERE nodeId IN (SELECT nodeId FROM cmsMember)
+	DELETE FROM cmsMember2MemberGroup
+	DELETE FROM umbracoContent WHERE nodeId IN (SELECT nodeId FROM cmsMember)
+	DELETE FROM cmsMember
+	DELETE FROM umbracoNode where nodeObjectType IN ('39EB0F98-B348-42A1-8662-E7EB18487560', '366E63B9-880F-4E13-A61C-98069B029728')
+    ```
+
+* In your Umbraco 10 database, run the following queries and make a note of the results as you will need them for later queries:
+
+   ```
+   SELECT MAX(id) FROM umbracoNode
+   SELECT MAX(id) FROM umbracoContentVersion
+   SELECT MAX(id) FROM umbracoPropertyData
+   ```
```

Comment:
This query should be run on the local database that we have taken from the v8 project and used on our v10 project right?

Otherwise two out of the three values are just gonna be NULL as our v10 DB is empty on the cloud environment.

Reply:
Good point, it does need to be clearer. Reading the whole guide there are at least three databases in play - v8 on Cloud, v8 upgraded to 10 locally, and v10 on Cloud. 

The local one may or may not have members already depending on whether it was created from a backup or by cloning down content (Step 1, first bullet), so actually the right source database is v8 on Cloud, the right target database is v10 on Cloud, and the right time is within Step 5, after "Transfer Content and Media from the local clone to the Development environment." and before "Test everything in the Development environment." Do you agree?